lib/chef/knife/core/bootstrap_context.rb

```diff
@@ -42,7 +44,11 @@ def bootstrap_environment
         end
 
         def validation_key
-          IO.read(File.expand_path(@chef_config[:validation_key]))
+          if File.exist?(File.expand_path(@chef_config[:validation_key]))
+            IO.read(File.expand_path(@chef_config[:validation_key]))
```

Comment:
What do people think about an info level log here to say that bootstrap is using the local validation_key?  I ran `knife configure` and I think it left my `validation_key` in my `knife.rb`.  If I use the new register-node-locally logic, I probably don't want that accidently sent to bootstrapped nodes.